extensions/amp-analytics/0.1/vendors.js

```diff
@@ -89,14 +89,23 @@ export const ANALYTICS_CONFIG = {
         'g3=${sponsorName}&' +
         'g4=${contentType}&' +
         'x=${scrollTop}&' +
+        'y=${scrollHeight}&' +
         'w=${screenHeight}&' +
```

Comment:
Thanks for flagging this; we're actually using 'w' for windowHeight, which is consistent with screenHeight for the purposes of mobile devices, but we're going to remove it for now.